client/src/components/Admin/Login/LoginForm.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import Input from './input';
+import './loginform.css';
+
+ class LoginForm extends Component {
+
+    state={
```

Comment:
use the useState hook for the functional component! it is as easy as setState!

Reply:
Use of hooks should be encouraged but this looks fine to me anyways.